_layouts/syllabus.html

```diff
@@ -55,6 +55,11 @@
       <a href="lectures/{{ page.slidesName }}.pdf">[pdf]</a>
       </p>
     {% endif %}
+    {% for oh_plus_plus in page.ohPlusPlus %}
+      <p>
+      <a href="/static/w17/oh-plus-plus/{{ oh_plus_plus }}">OH++ Slides</a>
```

Comment:
Might also be worth adding a link to the pretty GitHub rendering:

https://github.com/c4cs/c4cs.github.io/blob/master/static/SEMESTER/oh-plus-plus/wkXX-ohpp.md